### PR TITLE
APEXMALHAR-2480 Move Amazon S3 related examples from datatorrent examples to apex-malhar examples.

### PR DESCRIPTION
**Pull request by prasannapramod**

### CODE REVIEW

**Comment by prasannapramod** [comment]
@tweise @amberarrow @ashwinchandrap please review

**Comment by prasannapramod** [comment]
@tweise can you see

**Comment by prasannapramod** [comment]
@tweise I updated the PR title and commit message. Please see

**Comment by prasannapramod** [comment]
@amberarrow @tweise please see.

**Comment by prasannapramod** [comment]
@devtagare @tweise please see

**Comment by devtagare** [comment]
@prasannapramod  - LGTM

File: examples/s3/src/main/resources/META-INF/properties-s3TupleOutput.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <name>dt.operator.lineInput.prop.maxReaders</name>
+     <value>8</value>
+  </property>
+  <property>
+    <name>dt.operator.lineInput.prop.minReaders</name>
+     <value>1</value>
+  </property>
+  <property>
+    <name>dt.loggers.level</name>
+    <value>org.apache.apex.malhar.lib.fs.s3.*:DEBUG,org.apache.apex.*:DEBUG,com.datatorrent.stram.plan.physical.*:DEBUG,com.datatorrent.lib.*:DEBUG</value>
```
Comment (tweise) [inline examples/s3/src/main/resources/META-INF/properties-s3TupleOutput.xml]: Why is this required?

**Comment by prasannapramod** [comment]
@devtagare @tweise @ashwinchandrap please see
